newIDE/app/src/InstancesEditor/index.js

```diff
@@ -484,11 +484,21 @@ export default class InstancesEditor extends Component<Props> {
     // to protect against renders after the component is unmounted.
     this._unmounted = true;
 
-    this.selectionRectangle.delete();
-    this.instancesRenderer.delete();
-    this._instancesAdder.unmount();
-    this.pinchHandler.unmount();
-    this.longTouchHandler.unmount();
+    if (this.selectionRectangle) {
+      this.selectionRectangle.delete();
```

Comment:
For some reason, we see a few crashes when calling `.delete()` on something null. I'm not sure how this unmounting is called when this is null? (is it called twice?) (is it called too soon and the constructor has not finished?)...
Doing those checks to prevent this crash

Reply:
Weird but ok